### PR TITLE
Adjust mobile view

### PR DESCRIPTION
**Pull request by hcorson-dosch-usgs**
This MR primarily fixes an issue with the mobile view - there was overflow in the References section that was making the remainder of the page content float within a too-wide window:

On mobile the view could be panned horizontally:

image

It also address a few other small issues I noticed along the way

### Changes
* Wrap References text and allow for word breaks, so that content doesn't overflow
* Set max-width and max-height for inset map, since overflowed width in some tall and narrow windows
* Adjust quote box positioning so that is placed set distance from bottom of page. This makes the placement feel more consistent and fixes an issue where the 1950s drought box abutted the bottom of the page on a laptop-sized screen
* Fix a typo in the references hyperlink to 'What is Streamflow Drought'
* Drop the drought images js file and references to it in DroughtHistory.vue, since those images are no longer used
* Add a global vertical margin variable that is used to transform content within the dynamic svg so that the first y-axis label isn't cut off, which was happening on some window sizes. To keep positioning and alignment consistent, this same variable is used to transform the static svg and to set a top margin for the chart image.
* Drop body font size by 0.5 pt, so that scroll buttons don't wrap to two lines on laptop view on Edge
* Add bottom margin to scroll buttons on desktop, in case they do wrap
* Fix tense in intro - was mixing past and present tense
* Give intro a bit more breathing room by adding top and bottom padding
* Adjust the annotation-container-desktop grid so that drought title and text don't overlap if title wraps

### To test
pull changes locally and run `npm run serve`

### CODE REVIEW

File: src/assets/text/referencesText.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default {
         num: "1",
         authors: "Archer, A.A., Corson-Dosch, H, and Nell, C. (2023)",
         title: "What is streamflow drought?",
-        link: "https://labs.waterdata.usgs.gov/visualizations/what-is-drought/index.html#/L",
```
Comment (aaarcher-usgs) [inline src/assets/text/referencesText.js:10]: Nice catch!

File: src/components/DroughtHistory.vue [hunk 2]
```diff
@@ -453,7 +473,7 @@ export default {
           .range([0, this.overlayWidth])
 
         // Add scroll to elements (only used for scroll navigation)
-        const scrollToSpot = this.svgChartDynamic.selectAll('scrollToSpot')
+        const scrollToSpot = chartGroup.selectAll('scrollToSpot')
```
Comment (aaarcher-usgs) [inline src/components/DroughtHistory.vue:476]: I like this name change!

**Comment by aaarcher-usgs** [comment]
Suggested addition: Add link to the CASC regions in the regional description. https://www.usgs.gov/programs/climate-adaptation-science-centers 

`now they are shown by <a href="https://www.usgs.gov/programs/climate-adaptation-science-centers" target="_blank">Climate Adaptation Science Center</a> regions`

**Review by aaarcher-usgs — APPROVED** [review]
This looks great! Builds locally and does a nice job stabilizing the spacing and performance throughout. 

Screenshots showing fixed issue on mobile and better spacing with small-screen in Chrome: 

image

image

**Comment by hcorson-dosch-usgs** [comment]
@aaarcher-usgs  - one more small change to the regional grid layout on laptop-sized screen:

Previously:
image


Now:
image

**Pull request merged**